app/src/main/java/org/mozilla/fenix/nimbus/NimbusExperimentsFragment.kt

```diff
@@ -5,81 +5,119 @@
-    override fun onDestroyView() {
-        super.onDestroyView()
-        // Letting go of the resources to avoid memory leak.
-        adapter = null
-    }
-
-    private fun bindRecyclerView(view: View) {
-        val experimentsView = NimbusExperimentsView(
-            navController = findNavController(),
-        )
-
-        val recyclerView = view.findViewById<RecyclerView>(R.id.nimbus_experiments_list)
-        recyclerView.layoutManager = LinearLayoutManager(requireContext())
-
-        val shouldRefresh = adapter != null
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View = ComposeView(requireContext()).apply {
 
-        lifecycleScope.launch(IO) {
-            try {
+        setContent {
+            FirefoxTheme {
                 val experiments =
                     requireContext().components.analytics.experiments.getAvailableExperiments()
 
-                lifecycleScope.launch(Main) {
-                    runIfFragmentIsAttached {
-                        if (!shouldRefresh) {
-                            adapter = NimbusExperimentAdapter(
-                                experimentsView,
-                                experiments,
+                NimbusExperiments(
+                    availableExperiments = experiments,
+                    onSelectedExperiment = { selectedExperiment ->
+                        val directions =
+                            NimbusExperimentsFragmentDirections.actionNimbusExperimentsFragmentToNimbusBranchesFragment(
+                                experimentId = selectedExperiment.slug,
+                                experimentName = selectedExperiment.userFacingName,
                             )
-                        }
 
-                        view.findViewById<TextView>(R.id.nimbus_experiments_empty_message).isVisible =
-                            experiments.isEmpty()
-                        recyclerView.adapter = adapter
+                        findNavController().navigate(directions)
                     }
+                )
+            }
+        }
+    }
+
+    /**
+     * The list for the Nimbus Experiments,
+     *
+     * @param availableExperiments List of available items
+     * @param onSelectedExperiment Callback for when item is selected.
+     */
+    @Composable
+    fun NimbusExperiments(
+        availableExperiments: List<AvailableExperiment> = listOf(),
+        onSelectedExperiment: (AvailableExperiment) -> Unit
+    ) {
+        LazyColumn (
+            modifier = Modifier
+                .fillMaxSize()
+        ) {
+            items(availableExperiments) { experiment ->
+                NimbusExperimentItem(item = experiment, onSelectedExperiment = onSelectedExperiment)
+            }
+        }
+    }
+    /**
+        * Nimbus experiment item
+        *
+        * @param item current experiment
+    */
+    @Composable
+    fun NimbusExperimentItem(
```

Comment:
We have a `ListItem.kt` that has a variety of list item components designed by our Design System team. I think it makes sense to reuse this component for the experiment items.